src/components/prove-identity-welcome/index-existing-session.njk

```diff
@@ -10,7 +10,6 @@
     <h1 class="govuk-heading-l">{{ 'pages.proveIdentityWelcome.header' | translate }}</h1>
 
     <p class="govuk-body">{{ 'pages.proveIdentityWelcome.section1.paragraph1' | translate }}</p>
-    {% if supportLanguageCY %}
         {% set insetTextInnerHtml %}
```

Comment:
Minor but could we reformat the lines that were within the if now that they don't need the additional indenting

Reply:
(same for other template changes)